### PR TITLE
Update Typstry.jl to v0.3.0 (#55)

### PR DESCRIPTION
**Pull request by asinghvi17**

### CODE REVIEW

File: src/rendering/typst.jl [hunk 1]
```diff
@@ -38,9 +38,12 @@ function compile_typst(document::AbstractString)
             err = Pipe()
 
             try
-                # `pipeline` is not yet supported for `TypstCommand`
+                # `pipeline` is not yet supported for `TypstCommand`.
+                # We need to add Julia Mono as a typst font, but don't want to override user env specs.
```
Comment (jakobjpeters) [inline src/rendering/typst.jl:42]: Oh good catch! This got me to fix the same issue in a few methods for v0.4 :)

**Comment by jakobjpeters** [comment]
Typstry.jl v0.4.0 is out now too, primarily to support the latest Typst v0.12 :)

**Comment by asinghvi17** [comment]
Thanks for the heads up!  I'll register this now and then look at Typstry v0.4 compat.

**Comment by jakobjpeters** [comment]
Cool! I bumped the minimum Julia version to v1.10, and will continue to support that version as it's the LTS. LaTeXStrings.jl is set at v1.1, since `show(::LaTeXString)` errors on LaTeXStrings.jl v1.0 and Julia v1.10.

**Pull request merged**